__tests__/comments.test.js

```diff
@@ -61,4 +53,78 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(response.body.msg).toBe('Bad request');
         });
     });
+})
+
+describe('POST /articles/:article_id/comments', () => {
```

Comment:
An additional test checking that unnecessary properties are ignored on the request object would be great to see!